oap-mail/src/main/java/oap/mail/attachment/AttachmentUtility.java

```diff
@@ -0,0 +1,23 @@
+package oap.mail.attachment;
+
+import lombok.extern.slf4j.Slf4j;
+import oap.mail.Attachment;
+
+import javax.activation.DataHandler;
+import javax.mail.Part;
+import javax.mail.internet.MimeBodyPart;
+import java.net.URL;
+
+@Slf4j
+public class AttachmentUtility {
```

Comment:
There already is a utility class called Attachments. It has "inheritance like" architecture for multiparts. It should be either implemented the same way as with files or the one with files should be implemented as static methods. 

there is a method called makeMimeType handling different i18n cases - should be used. See MimeFileDataSource

Reply:
fixed